Source/WebKit/UIProcess/WebProcessPool.cpp

```diff
@@ -1097,7 +1097,7 @@ void WebProcessPool::processDidFinishLaunching(WebProcessProxy& process)
     if (m_memorySamplerEnabled) {
         SandboxExtension::Handle sampleLogSandboxHandle;        
         WallTime now = WallTime::now();
-        auto sampleLogFilePath = makeString("WebProcess", static_cast<unsigned long long>(now.secondsSinceEpoch().seconds()), "pid", process.processID());
+        auto sampleLogFilePath = makeString("WebProcess"_s, static_cast<unsigned long long>(now.secondsSinceEpoch().seconds()), "pid"_s, process.processID());
```

Comment:
I wonder if we could write this as `now.secondsSinceEpoch().secondsAs<uint64_t>()` instead?